specs-go/config.go

```diff
@@ -33,6 +33,34 @@ type Spec struct {
 	ZOS *ZOS `json:"zos,omitempty" platform:"zos"`
 }
 
+// Scheduler represents the scheduling attributes for a process. It is based on
+// the Linux sched_setattr(2) syscall.
+type Scheduler struct {
+	// Policy represents the scheduling policy (e.g., SCHED_FIFO, SCHED_RR, SCHED_OTHER).
+	Policy LinuxSchedulerPolicy `json:"policy"`
+
+	// Nice is the nice value for the process, which affects its priority.
+	Nice int32 `json:"nice"`
```

Comment:
If you know, please let me know, does this apply to process groups?:pray: Or only one process?

Reply:
from my understanding that is only for one process or thread:

Reply:
I agree with you. I thought the user would expect the nice value to be applied to all processes in the container? What do you think?

Reply:
How about specifying whether it applies to the container init process alone or to all processes in the container?